### PR TITLE
MSAL Major Version Upgrades April 2023

### PR DESCRIPTION
**Pull request by hectormmg**

### CODE REVIEW

**Review by shoatman — APPROVED** [review]
lgtm - Thanks Hector

**Review by jo-arroyo — APPROVED** [review]
LGTM!

File: .beachballrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "branch": "dev",
+    "branch": "msal-browser-v3",
```
Comment (tnorling) [inline .beachballrc]: Let's revert this change as we move this into dev

File: .github/workflows/beachball-check.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:  
       - dev
+      - msal-browser-v3
```
Comment (tnorling) [inline .github/workflows/beachball-check.yml]: Same here

File: lib/msal-angular/package.json [hunk 3]
```diff
@@ -33,23 +33,21 @@
   "main": "./dist/bundles/azure-msal-angular.umd.js",
   "typings": "./dist/azure-msal-angular.d.ts",
   "beachball": {
-    "disallowedChangeTypes": [
-      "major"
```
Comment (tnorling) [inline lib/msal-angular/package.json:37]: Let's restore the disallowedChangeTypes everywhere as well

Comment (tnorling) [reply lib/msal-angular/package.json:37]: Actually this can wait until we release, but let's make sure this is tracked, if not already, so we don't forget

File: lib/msal-node/docs/response.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Response
+
+MSAL will return an `AuthenticationResult.ts` object as a response to all acquire token APIs:
+
+#### `msal-browser` public APIs for token acquisition:
```
Comment (tnorling) [inline lib/msal-node/docs/response.md]: This doc seems like it's in the wrong place...

File: package.json [hunk 5]
```diff
@@ -60,9 +60,9 @@
     "lint:react": "eslint lib/msal-react/src --ext .ts --ext .tsx",
     "clean": "lerna run clean --no-private && lerna clean",
     "clean:coverage": "rimraf ./.nyc_output/*",
-    "beachball:check": "beachball check --branch origin/dev",
-    "beachball:change": "beachball change --branch origin/dev",
-    "beachball:bump": "beachball bump --branch origin/dev --bumpDeps false && node release-scripts/updateVersion.js",
+    "beachball:check": "beachball check --branch origin/msal-browser-v3",
```
Comment (tnorling) [inline package.json]:
```suggestion
    "beachball:check": "beachball check --branch origin/dev",
```

File: package.json [hunk 6]
```diff
@@ -60,9 +60,9 @@
...
+    "beachball:change": "beachball change --branch origin/msal-browser-v3",
```
Comment (tnorling) [inline package.json]:
```suggestion
    "beachball:change": "beachball change --branch origin/dev",
```

File: package.json [hunk 7]
```diff
@@ -60,9 +60,9 @@
...
+    "beachball:bump": "beachball bump --branch origin/msal-browser-v3 --bumpDeps false && node release-scripts/updateVersion.js",
```
Comment (tnorling) [inline package.json]:
```suggestion
    "beachball:bump": "beachball bump --branch origin/dev --bumpDeps false && node release-scripts/updateVersion.js",
```

**Review by tnorling — COMMENTED** [review]
A couple minor housekeeping things, otherwise looks good

File: lib/msal-angular/package.json (hunk 3 above)
Comment (hectormmg) [reply lib/msal-angular/package.json:37]: Yes, it's tracked on ADO.
